### ISSUE TITLE
diff of the configuration at pretrain stage between the 34B model and 8B model

### ISSUE THREAD

**Issue by royzhang12** [OpenGVLab/InternVL#683]
Hi @czczup,

Thanks for providing the wonderful job. I just have some questions about the training config of internvl-chat at the pretrain stage.

I noticed that the configuration for training the 34B model and the 8B model is quite different.
<img width="1281" alt="image" src="https://github.com/user-attachments/assets/4689f743-1749-4b25-bd0b-c7714ea88c72">
(internvl-chat 1.5 pretrain config: left: 34B model; right: 8B model)

Regarding the differences, I have the following questions:
1. `freeze_backbone`: Why is the vit frozen in the 34B model but not in the 8B model?
2. Learning rate & weight decay: Did you make this decision because of the different batch sizes or because you froze the vit?
3. Drop path rate: Why are the drop path rates different?
4. Mlp path: Why reinitialize the mlp using the internvl 1.2?

Looking forward to your response, thanks a lot 😁

**Comment by royzhang12** [comment]
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/68cfa6af-7766-4d99-b9ca-e906aa23c5a6">
Similar Diff between(34B:left model and 20B model:right)

**Comment by royzhang12** [comment]
@czczup  大佬可以辛苦抽空帮忙解答下吗？🥹

**Comment by royzhang12** [comment]
捞一捞,辛苦大佬帮忙看下 @czczup
